tests/foreman/cli/test_host.py

```diff
@@ -1635,6 +1636,10 @@ def restore_sca_setting():
             "subscriptions": [{"id": custom_sub.id, "quantity": 1}],
         }
     )
+    # make sure repo is enabled
+    katello_host_tools_host.enable_repo(
+        f'{module_org.name}_{prod.name}_{custom_repo.name}', force=True
+    )
```

Comment:
GR template adds a custom repo already in /etc/yum.repos.d/ which will be enabled by default, so this step looks redundant

Reply:
@pondrejk yes, but this enable_repo func uses subscription-manager to enable this repos, and this custom repos with same content is already enabled from GR templates